README.md

```diff
@@ -14,406 +14,61 @@ This repository contains all specification files used to generate code for the R
-empowers a developer to write a full Rosetta Data API server by only implementing an interface.
-This package automatically validates client requests and calls the functions you implement
-with pre-parsed requests (instead of in raw JSON).
+* [Asking Questions](CONTRIBUTING.md/#asking-questions)
+* [Providing Feedback](CONTRIBUTING.md/#providing-feedback)
+* [Reporting Issues](CONTRIBUTING.md/#reporting-issues)
 
-There is a simple [example](https://github.com/coinbase/rosetta-sdk-go/tree/master/examples/server) of
-how to write an implementation using this package in [rosetta-sdk-go](https://github.com/coinbase/rosetta-sdk-go).
+Read our [Contributing](CONTRIBUTING.MD) documentation for more information.
 
-#### Using Another Language
-If you plan to use a language other than Golang, you will need to either codegen
-a server (the [overview](#overview) mentions some tools that can help with this) or
-write one from scratch. If you do choose to write an implementation in another language,
-we ask that you create a separate repository in an SDK-like format for all the code you generate
-so that other developers can use it (see the note on [SDKs in more languages](#sdks-in-more-languages)). Check out
-[rosetta-sdk-go](https://github.com/coinbase/rosetta-sdk-go) for an example of how to generate
-code from this specification.
+When you've finished an implementation for a blockchain, share your work in the [ecosystem category of the community site](https://community.rosetta-api.org/c/ecosystem). Platforms looking for implementations for certain blockchains will be monitoring this section of the website for high-quality implementations they can use for integration. Make sure that your implementation meets the [expectations](https://www.rosetta-api.org/docs/node_deployment.html) of any implementation.
 
-### CurveTypes and SignatureTypes
-The Construction API only supports detached, curve-based key generation and signing.
-In short, this means that Construction API implementations will never have
-access to private keys and will only ever interact with public keys and signatures
-in standardized formats. These formats are listed below:
+## Related Projects
 
-_CurveType and SignatureType are purposely decoupled as a curve could be used
-with multiple signature types (i.e. `secp256k1:ECDSA` and `secp256k1:Schnorr`)._
+* [rosetta-sdk-go][https://github.com/coinbase/rosetta-sdk-go] — The `rosetta-sdk-go` SDK provides a collection of packages used for interaction with the Rosetta API specification. Much of the SDK code is generated from this, the [rosetta-specifications](https://github.com/coinbase/rosetta-specifications) repository.
+* [rosetta-cli][https://github.com/coinbase/rosetta-ecosystem] — Use the `rosetta-cli` tool to test your Rosetta API implementation. The tool also provides the ability to look up block contents and account balances.
 
-#### Supported CurveTypes
```

Comment:
Where are all these docs moving to?

Reply:
They would move to rosetta-api.org. Just not sure where in there to put them.